src/panels/lovelace/cards/hui-entity-card.ts

```diff
@@ -122,7 +125,15 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
           </div>
         </div>
         <div class="info">
-          <span class="value"
+          <span
+            class="value"
```

Comment:
So we are giving the text color here? Not sure if we should, yellow in white will have a bad contrast for instance. I would rather color the icon or the graph.

Reply:
Using state_color options like I suggested above could get us around that as it would be user defined. Like the idea of the graph being the one to color for the sensor card. I could go either way on icon vs text, especially if we do state_color options like in my other PR.